### DISCUSSION TITLE
Orbit B-hyve 94990, is it compatible or...?

### DISCUSSION THREAD

**Discussion by GileanEU** [sebr/bhyve-home-assistant#118] [Q&A]
I bought Orbit B-hyve 94990 because I found this component and thought I could get this device to show up in Home Assistant with it... But I am not sure if it's because of incompatible equipment or something else, but I cannot see any entities named bhyve or orbit or anything like that. I have added this on my configuration.yaml after installing this component and restarted few times:

```
bhyve:
  username: !secret bhyve_username
  password: !secret bhyve_password

sensor:
  - platform: bhyve

switch:
  - platform: bhyve

binary_sensor:
  - platform: bhyve
```

The only thing I can find from logs is: "Setup of bhyve is taking over 10 seconds." and some time later:

```
Logger: homeassistant
Source: custom_components/bhyve/pybhyve/websocket.py:57
Integration: Orbit BHyve Integration ([documentation](https://github.com/sebr/bhyve-home-assistant/blob/master/README.md))
First occurred: 15.59.18 (1 occurrences)
Last logged: 15.59.18

Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/config/custom_components/bhyve/pybhyve/websocket.py", line 57, in _ping
    await self._ws.send_str(json.dumps({"event": "ping"}))
  File "/usr/local/lib/python3.9/site-packages/aiohttp/client_ws.py", line 151, in send_str
    await self._writer.send(data, binary=False, compress=compress)
  File "/usr/local/lib/python3.9/site-packages/aiohttp/http_websocket.py", line 690, in send
    await self._send_frame(message, WSMsgType.TEXT, compress)
  File "/usr/local/lib/python3.9/site-packages/aiohttp/http_websocket.py", line 601, in _send_frame
    raise ConnectionResetError("Cannot write to closing transport")
ConnectionResetError: Cannot write to closing transport
```

**Comment by sebr** [comment]
Hi @GileanEU - the faucet/tap timer with the wifi hub is indeed supported. Looks like something else has gone wrong, but not sure. I have never experienced or had this reported to me before.

I would suggest that you [enable debug logging](https://github.com/sebr/bhyve-home-assistant#debugging), restart your HA instance and then revert back with updated logs.

**Comment by GileanEU** [comment]
Thank you, but I tried restarting HA several times, and there was no bhyve -related entities available at all, anywhere. In the end I decided to return my 94990 (did that earlier today), because it was new and I bought it for HA integration only. And I decided to go with solenoid valve + smart plug -route instead, as I have weather system integrated to HA already, so I can build automations that way too.
